### PR TITLE
Remove all selfbot related checks

### PR DESCRIPTION
**Pull request by KingDGrizzle**
### Description of the PR
discord.js has [this](https://github.com/discordjs/discord.js/pull/2559) selfbots are getting the axe so I decided to shoo them away with this PR.

### Changes Proposed in this Pull Request (List new items in CHANGELOG.MD)

- Removes everything that checks if you are on a selfbot

### Semver Classification

- [ ] This PR only includes documentation or non-code changes.
- [ ] This PR fixes a bug and does not change the (intended) framework interface.
- [ ] This PR adds methods or properties to the framework interface.
- [x] This PR removes or renames methods or properties in the framework interface.

### CODE REVIEW

**Comment by bdistin** [comment]
CommandHandler (monitor) lines 104-105

**Comment by bdistin** [comment]
ignoreSelf is already true

**Comment by bdistin** [comment]
awaiting d.js removing userbot support

**Comment by UnseenFaith** [comment]
https://github.com/dirigeants/klasa/blob/master/src/lib/util/util.js#L57-58
This also needs to be removed with this PR

File: src/lib/util/util.js [hunk 1]
```diff
@@ -54,8 +54,6 @@ class Util {
 	static initClean(client) {
```
Comment (kyranet) [inline src/lib/util/util.js]:
```javascript
static initClean(client) {
	sensitivePattern = new RegExp(Util.regExpEsc(client.token), 'gi');
}
```

**Pull request merged**